docker-compose.yml

```
@@ -23,6 +23,8 @@ services:
       - ENABLE_RBAC=${ENABLE_RBAC:-0}
       - LESS_LOG_NOISE=1
       - PERCONA_TEST_VERSION_SERVICE_URL=${PERCONA_TEST_VERSION_SERVICE_URL}
+      - PMM_WATCHTOWER_HOST=${PMM_WATCHTOWER_HOST:-http://watchtower:8080}
+      - PMM_WATCHTOWER_TOKEN=${PMM_WATCHTOWER_TOKEN:-123}
```

```suggestion
      - PMM_WATCHTOWER_TOKEN=${PMM_WATCHTOWER_TOKEN:-INSECURE_TOKEN}
```
I figured we can be more explicit not to promote the use of default insecure tokens.